### DISCUSSION TITLE
Do i need to print all the files?

### DISCUSSION THREAD

**Discussion by AndiiCodes** [martinwoodward/octolamp#33] [General]
Hi, thanks for this amazing good looking project, i dont own a 3d printer so i'm using my uni 3d printer.. i was wondering if i need to print all the files? it might sound like a dump question , but do i need the outer black part? Thanks again for this project!!

**Comment by martinwoodward** [comment]
Hey @AndiiCodes - you'll need the following at a minimum:
- Black outer
- White reflector
- White inner

The components snap-fit together to hold each other in place along with the electronics.

Good luck with the build - be sure to send us photos when you have it!!!
